resources/lib/ui/http.py

```diff
@@ -31,6 +32,11 @@ def Session():
         })
         _SESSION = s
 
+        token = TokenDecoder.get_token(lambda x: __send_request(s, x, None, None, False))
```

Comment:
you just have to make sure that the token gets requested after the cloudflare check but before other requests so it can be appended as cookie to all requests afterwards. (to be on the safe side for further changes)